paasta_tools/firewall.py

```diff
@@ -50,48 +57,95 @@ def chain_name(self):
         return chain
 
     @property
+    def config(self):
+        load_fn = {
```

Comment:
This kinda duplicates
https://github.com/Yelp/paasta/blob/8eb6d8b406b10be540a825152341439a999e95cb/paasta_tools/cli/utils.py#L665